packages/liquidator/test/Liquidator.js

```diff
@@ -2028,6 +2028,22 @@ contract("Liquidator.js", function(accounts) {
             "Correctly deals with reserve being the same as collateral currency using DSProxy",
             async function() {
               // create a new liquidator and set the reserve currency to the collateral currency.
+              const proxyTransactionWrapper = new ProxyTransactionWrapper({
```

Comment:
this test was somewhat working correctly before but in an unintended way. effectively the `proxyTransactionWrapper` was not correctly initialized and was still using a reserve currency( not the collateral currency). The test below _does_ validate that shortfall behavior works as expected but in this test it was shortfalling on reserve (not also collateral). 

After fixing this it correctly values a shortfall in _both_ collateral and reserve.